security/yubi/windows/README.md

```diff
@@ -0,0 +1,63 @@
+# Windows Setup
+In general, the usability of a Yubikey in Windows is not as easy as it should be. But worry not! We've got you covered. This guide includes everything necessary to setup your environments to work with your hardware-key seamlessly (gpg-wise at least).
+
+## What
+This setup enables you to setup your gpg agents in a way, so they can make use of gpg subkeys stored on your smartcard in an effortless manner.
+After completing this setup you'll be able to use your smartcard as the single source of gpg keys in both, your Windows OS as well as WSL2 distributions installed on it. Here we will utilize 2 different kinds of pageants:
+* https://github.com/benpye/wsl-ssh-pageant - to setup your host OS (Windows itself)
+* https://github.com/BlackReloaded/wsl2-ssh-pageant - to setup your guest OS in WSL2
+
+## How
+There's 2 ways doing this:
+- The autoamted way -> a script which copies configs all around the place
+- The manual way -> copying the configs yourself (as fallback, if above's script doesn't work)
+Both need a restart of your OS at the end to confirm, that everything works out-of-the-box.
+
+### Requirements
+
+* Windows 10 or higher
+* Windows Powershell (should be a no brainer)
+* Local Admin permissions on your machine
+* As of now you still have to have the `chocolatey` package manager installed [from here](https://chocolatey.org/install).
```

Comment:
I would recommend to rewrite this section / scripts to use winget since it is the new package manager for windows which is preinstalled on later windows 11 versions and can be installed on windows 10 as well.